lib/xapi/problem.rb

```diff
@@ -18,7 +18,7 @@ def id
     end
 
     def git_url
-      Track.new(path, track_id).repository << '/tree/master/' << slug
+      @repository << '/tree/master/' << slug
```

Comment:
Did you mean to store this in `@repository`? If you call git_url twice in a row, what does it look like?